### PR TITLE
add access token authentication function

### PR DESCRIPTION
**Pull request by yahtoo**

### CODE REVIEW

File: blockchain/accesstoken/accesstoken.go [hunk 1]
```diff
@@ -27,147 +26,132 @@ var (
 	ErrDuplicateID = errors.New("duplicate access token ID")
 	// ErrBadType is returned when Create is called with a bad type.
 	ErrBadType = errors.New("type must be client or network")
+	// ErrNoMatchID is returned when Delete is called on nonexisting ID.
+	ErrNoMatchID = errors.New("nonexisting access token ID")
```
Comment (liuchengxu) [inline blockchain/accesstoken/accesstoken.go]: FYI, `"nonexisting access token ID"` -> `no matching token ID`.

File: blockchain/accesstokens.go [hunk 2]
```diff
@@ -2,111 +2,54 @@ package blockchain
...
-/*
-const (
-	defGenericPageSize = 100
-)
-*/
 var errCurrentToken = errors.New("token cannot delete itself")
 
-func (bcr *BlockchainReactor) createAccessToken(ctx context.Context, x struct{ ID, Type string }) (*accesstoken.Token, error) {
-	token, err := bcr.accesstoken.Create(ctx, x.ID, x.Type)
+func (a *BlockchainReactor) createAccessToken(ctx context.Context, x struct{ ID, Type string }) interface{} {
```
Comment (liuchengxu) [inline blockchain/accesstokens.go]: Why don't use `bcr` for `BlockchainReactor`?

File: cmd/cobra/commands/accesstoken.go [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+import (
+	"context"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	jww "github.com/spf13/jwalterweatherman"
+)
+
+var accessTokenCreatCmd = &cobra.Command{
```
Comment (liuchengxu) [inline cmd/cobra/commands/accesstoken.go]: The current convention is name a variable by its command accordingly, e.g., `create-access-token` -> `createAccessTokenCmd`.

File: blockchain/accesstokens.go (hunk 2 above)
Comment (Blockmeta) [reply blockchain/accesstokens.go]: bcr is better

**Pull request closed without merging**